### ISSUE TITLE
[Anon] Bug report: Astra Militarum - Codex.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-7th-edition#1641]
**File:** Astra Militarum - Codex.catz

**BattleScribe version:** 1.15.07

**Platform:** Android

**Dropbox:** No

**Description:** Astra codex selection dose not pull up force organization with the 
 Cadian battle group will not pull up roster when selected for Astra militarum codex

**Comment by KealL** [comment]
The entries for the detachment have not yet been made. If not assigned by Thursday I will look at.

(also appears that the entry in the game system are all abit mixed up for detachments)

**Comment by ecrousseau** [comment]
I have created a catalogue file for the Cadian Battle Group - just waiting on permission to commit it.

**Comment by KealL** [comment]
As in a separate catalog, independent of the AM one?

**Comment by amis92** [comment]
Would that be you too? https://www.reddit.com/r/BattleScribe/comments/3v8j0n/ive_created_a_catalogue_file_for_the_cadian/

As KeaIL asked: if it's new catalogue, separate from the existing one, we'll probably not consider it. These options will have to be incorporated into the main file. Of course, the last saying goes to maintainers, which currently looks to be @axisofentropy .

**Comment by amis92** [comment]
Okay, closing it as a duplicate of #1642.

**Comment by axisofentropy** [comment]
On 12/03/2015 03:33 PM, Amadeusz Sadowski wrote:

> Of course, the last saying goes to maintainers, which currently looks to
> be @axisofentropy https://github.com/axisofentropy .

Sorry I'm not working on that this week. I did move a few units into
Shared Units in preparation, but that's as far as I got. Go for it.
